### DISCUSSION TITLE
2.1.1

### DISCUSSION THREAD

**Discussion by T8RIN** [T8RIN/ImageToolbox#267] [Announcements]
## What's Changed
* ### App now named "Image Toolbox" in favour of better understanding it main functions, because now we not just resize images, we can almost everything
* Added ability to apply filter chains to images and also 50 filters included on the start
* Resize by limits option added
* Loading image from internet - now you can easily download some images from web
* Fix previewing large images
* Added option to compare output and start image
* Now you can hold and get tooltips on each resize type: Explicit, Flixible, Ratio
* Shared images preview like in android 12 sheet
* Bumped dependencies
* Fixed critical bugs
* Updated UI
* Added new emojis

Thanks for using Image Toolbox (Image Resizer) stay tuned for the next updates ❤️ 

**Full Changelog**: https://github.com/T8RIN/ImageToolbox/compare/2.0.5...2.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/T8RIN/ImageToolbox/releases/tag/2.1.1'>2.1.1</a>.</em>

**Comment by T8RIN** [comment]
Unfortunately, it has many bugs with filters applying, so i will release 2.1.2 today :(

**Comment by T8RIN** [comment]
And also, please, if something doesn't work, open an issue, and i will fix this, cuz i'm disappointed with the fact that this large release has been delayed so many times :(

**Reply by carkks** [reply]
No problem for me, a tool of incredible 12mb having these functions already helps a lot in everyday life.  Even if it takes time, I will continue to wait for updates!

**Reply by T8RIN** [reply]
Thanks!

**Comment by T8RIN** [comment]
I hope that it will be released tomorrow
